src/components/cesium_map/cesium_UI.js

```diff
@@ -119,9 +121,24 @@ function updatePrimitive(latitude, longitude) {
   oboePrimitive = setPrimitive.load({ lat: latitude, long: longitude, searchFields: searchFields, rows: 100000 });
   lat = latitude;
   long = longitude;
+}
 
+/**
+ * This function calculate the distance between two camera positions
+ *
+ * @param {*} lat1 old position latitude
+ * @param {*} lon1 old position longtitude
+ * @param {*} lat2 new position latitude
+ * @param {*} lon2 new position longtidue
+ * @returns distance in kilometer
+ */
+function distance(lat1, long1, lat2, long2) {
```

Comment:
As a way of making my life easier, I like to name functions/variables with their units in the name, e.g. `distanceInKm`.